src/main/java/org/jabref/Globals.java

```diff
@@ -58,7 +58,7 @@ private Globals() {
     }
 
     // Key binding preferences
-    public static KeyBindingRepository getKeyPrefs() {
+    public static synchronized KeyBindingRepository getKeyPrefs() {
```

Comment:
Refs [item 3 of effective java](https://github.com/mgp/book-notes/blob/master/effective-java-2nd-edition.markdown#item-3-enforce-the-singleton-property-with-a-private-constructor-or-an-enum-type), doesn't it?